netmanager/src/views/components/DataDisplay/DeviceView/DeviceEdit.js

```diff
@@ -162,6 +161,7 @@ const EditDeviceForm = ({ deviceData, siteOptions }) => {
               onChange={handleTextFieldChange}
               error={!!errors.phoneNumber}
               helperText={errors.phoneNumber}
+              placeholder="+256XXXXXXXXX"
```

Comment:
Just curious as to why we need placeholders when editing. Could you shed some light on this? @FaithDaka

Reply:
The phone number is not part of the attributes during the initial creation of a device. 
So during editing, the user needs to know the format in which accepted phone numbers are written otherwise they'll be frustrated with bad request errors.

Reply:
I didn't hit the comment button the first time I replied to this and was wondering why the PR was never merged.